seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/source/JdbcSourceFactory.java

```diff
@@ -155,11 +155,11 @@ static PartitionParameter createPartitionParameter(
                 max =
                         config.getPartitionUpperBound().isPresent()
                                 ? config.getPartitionUpperBound().get()
-                                : Long.parseLong(rs.getString(1));
+                                : rs.getLong(1);
```

Comment:
> If the table is empty, then rs.next() return false

no, if the table is empty, rs.next() return ture, may because of subquery

Reply:
I have test some empty table (c1 int , c2 bigint ,c3 tinyint)
if query is corretly, PartitionParameter#minValue will not throw NPE, because the getLong() return long ,it has default value 0

Reply:
![image](https://user-images.githubusercontent.com/35491928/231823057-ef83d2c8-f716-4c55-b10c-afbcfb3113ae.png)

![image](https://user-images.githubusercontent.com/35491928/231823093-305cf67f-a091-427f-9c28-ffa4e50c31b1.png)

Reply:
done, I refactor Long to long, it has default value if not set.
Besides, the property of PartitionParameter has no private added